extension/lib/main.js

```diff
@@ -113,14 +110,65 @@ exports.main = function (options, callbacks) {
         widget.tooltip = config.extension.widget_tooltips[data];
         break;
     }
+  });*/
+
+  var { Frame } = require("sdk/ui/frame");
+  var { ToggleButton } = require('sdk/ui/button/toggle');
+  var { Toolbar } = require("sdk/ui/toolbar");
+
+  var frame = new Frame({
+    url: "./frame/frame.html"
   });
 
+  var menuButton = ToggleButton({
+    id: "menu-button",
+    label: "Menu",
+    icon: {
+      "32" : "./addon32.png",
+      "64" : "./addon64.png"
```

Comment:
From the docs: https://developer.mozilla.org/en-US/Add-ons/SDK/Low-Level_APIs/ui_button_toggle#Constructors
I think you can add a 16px icon that will be used for items in the toolbar. If the icon is moved to the menu panel, it will automatically switch to the larger icon.